src/backend/db/posts.ts

```diff
@@ -7,22 +7,48 @@ import { formatDate } from "../utils/authUtils";
  * You can add default posts of your wish with different attributes
  * */
 
+export interface postType {
```

Comment:
suggestion: generally from the backend we only get the data/response, so defining the `interface` on frontend makes more sense